### ISSUE TITLE
Mounted filesystems not accessible by root

### ISSUE THREAD

**Issue by jandubois** [AkihiroSuda/lima#5]
I ran `limactl start default` without adding the hypervisor entitlements to qemu (see #4) and then connected via `lima bash`. The mounted directories `/tmp/lima` (and `/Users/jan`) work as expected for the normal user:

```console
jan@lima-default:/$ ls -l /tmp/
total 24
drwxr-xr-x 1 root root  192 May 15 00:55 lima
drwx------ 3 root root 4096 May 15 00:53 snap.lxd
drwx------ 2 jan  jan  4096 May 15 00:53 ssh-VWQD1U87aO
drwx------ 3 root root 4096 May 15 00:53 systemd-private-193c8c72bfc547b5bd03fbf7a3776fda-systemd-logind.service-BsE9tL
drwx------ 3 root root 4096 May 15 00:53 systemd-private-193c8c72bfc547b5bd03fbf7a3776fda-systemd-resolved.service-cfZvr4
drwx------ 3 root root 4096 May 15 00:53 systemd-private-193c8c72bfc547b5bd03fbf7a3776fda-systemd-timesyncd.service-fAE656
jan@lima-default:/$ touch /tmp/lima/foo
```

But the root users doesn't seem to have write/execute permission to them:

```console
jan@lima-default:/$ sudo ls -l /tmp/
ls: cannot access '/tmp/lima': Permission denied
total 20
d????????? ? ?    ?       ?            ? lima
drwx------ 3 root root 4096 May 15 00:53 snap.lxd
drwx------ 2 jan  jan  4096 May 15 00:53 ssh-VWQD1U87aO
drwx------ 3 root root 4096 May 15 00:53 systemd-private-193c8c72bfc547b5bd03fbf7a3776fda-systemd-logind.service-BsE9tL
drwx------ 3 root root 4096 May 15 00:53 systemd-private-193c8c72bfc547b5bd03fbf7a3776fda-systemd-resolved.service-cfZvr4
drwx------ 3 root root 4096 May 15 00:53 systemd-private-193c8c72bfc547b5bd03fbf7a3776fda-systemd-timesyncd.service-fAE656
jan@lima-default:/$ sudo touch /tmp/lima/foo
touch: cannot touch '/tmp/lima/foo': Permission denied
```

**Comment by AkihiroSuda** [comment]
I'm planning to replace sshfs with 9p (#1)
This issue will be probably resolved together at that time.

**Comment by AkihiroSuda** [comment]
This issue is fixed in Lima v0.2.1.
